src/Umbraco.Core/Persistence/Repositories/IDataTypeRepository.cs

```diff
@@ -14,4 +14,13 @@ public interface IDataTypeRepository : IReadWriteQueryRepository<int, IDataType>
     /// <param name="id"></param>
     /// <returns></returns>
     IReadOnlyDictionary<Udi, IEnumerable<string>> FindUsages(int id);
+
+    /// <summary>
+    ///     Returns a dictionary of content type <see cref="Udi" />s and the data type (List view) aliases that use a
+    ///     <see cref="IDataType" />
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+
+    IReadOnlyDictionary<Udi, IEnumerable<string>> FindListViewUsages(int id);
```

Comment:
This is a breaking change, anybody already implementing this interface will get a build error and plugins that haven't been updated will make the whole site error out.

Luckily there's an easy way to work around this and that is to introduce a default implementation, an example can be seen here: https://github.com/umbraco/Umbraco-CMS/pull/11867#issuecomment-1178869417